baekjoon_9/1978.py

```diff
@@ -0,0 +1,24 @@
+# 1978 - 소수 찾기
+
+n = int(input())
+
+sosu = list(map(int, input().split()))  # 숫자열 입력
+
+result = 0
+
+for i in sosu:  # 입력된 문자열에서 하나씩 꺼내 쓰면...
+
+    cnt = 0  # 소수 판별용 카운터, 소수는 나누어지는 값이 자기 자신 하나여야 한다!
+
+    if i == 1:  # 1은 소수가 아니다
+        pass
+
+    for j in range(2, i + 1):  # 2부터~  sosu 안의 값 i 까지
+
+        if i % j == 0:
```

Comment:
python은 `0`은 `False`, `0이외의 수`는 `True`로 판단해요
이렇게 바꿔도 잘 동작하겠죠?
```suggestion
        if not i % j:
```